app/components/listing/LocationTimesAccordion.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import { LocationDetails } from "models";
+import classNames from "classnames";
+import styles from "./LocationTimesAccordion.module.scss";
+
+/*
+    Accordion built per a11y guidelines:
+    https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
+*/
+
+const LocationTimesAccordion = ({
+  locationRenderer,
+  locations,
+}: {
+  locations: LocationDetails[];
+  locationRenderer: (loc: LocationDetails) => React.ReactElement;
+}) => {
+  const [activeIndices, setActiveIndices] = useState<number[]>([0]);
+
+  const toggleAccordion = (index: number) => {
```

Comment:
Something about using the loop array index as the lookup value in the _active_ set feels a little fragile. Mainly insofar as reasoning about this gets a little tricky because there are two arrays: 1) an ordered array for the rendered loop, and 2) the active array with unordered indices. Line 23 where your predicate function does `(i) => i !== index` is definitely a doozy. Naming could probably clear this up. But I also wonder if using another piece of location data would be less cognitively taxing (I'm imagining being tired and hungry and trying to figure this code out).

Reply:
I fixed this up!